packages/stylelint-config/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "@humanmade/stylelint-config",
+  "description": "Human Made coding standards for CSS and SCSS.",
+  "version": "0.6.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/humanmade/coding-standards.git"
+  },
+  "devDependencies": {
+    "stylelint-config-wordpress": "^13.0.0"
```

```suggestion
    "stylelint-config-wordpress": "^14.0.0"
```

p.s. v14 was released a couple of weeks ago https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/releases/tag/14.0.0

---

Good note, I'll handle that in a separate PR 👌